File: Java/Solutions/8.1. Exam Preparation - Part I/src/Bricks.java
import java.util.Scanner;

public class Bricks {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        int x = Integer.parseInt(scanner.nextLine());
        int w = Integer.parseInt(scanner.nextLine());
        int m = Integer.parseInt(scanner.nextLine());

        int bricksInOneCourse = w * m;

        double totalCourses = Math.ceil((double)x / bricksInOneCourse);

        System.out.print(totalCourses);
    }
}

File: JavaScript/Solutions/08.Exam Preparation - Part I/Bricks.js
function solve([x, w, m]) {
    x = Number(x);
    w = Number(w);
    m = Number(m);

    let bricksInOneCourse = w * m;

    let totalCourses = Math.ceil(x / bricksInOneCourse);

    console.log(totalCourses);
}

solve([120, 2, 30]);
solve([355, 3, 10]);
solve([5, 12, 30]);